ucp/_libs/ucp_py.pyx

```diff
@@ -59,6 +60,30 @@ def update_pending_messages():
     for msg in dones:
         PENDING_MESSAGES.pop(msg)
 
+def unfinished_messages_info():
+    """Returns info of all current unfinished messages"""
+    update_pending_messages()
```

Comment:
@madsbk I have a concern with the usage of `update_pending_messages()` in a function which is expected to report on the state of the communication channel. `update_pending_messages()` internally calls `msg.check()` which pings the progress engine and can result in communication progress. I think this is not intended for the purpose of reporting information about unfinished messages. What are your thoughts on this?

Reply:
I think you are completly right :)
Fixed in b212df0